server/comment/index.js

```diff
@@ -16,16 +15,15 @@ const defaultListQuery = {
   paranoid: false
 };
 
-router.get('/courses/:courseId/comments/subscribe', sse, channel.subscribe);
+router.get('/subscribe', sse, channel.subscribe);
```

Comment:
In all other router files `router.param` comes first. Is it really required to keep SSE `/subscribe` on top, can we put it bellow `commentId` param? :thinking:

Reply:
First I modified `subscribe`  function in `channel.js` 👇 , because we have `course` in request and there's nothing in body. Second, i think that `param` interrupts subscribe route because `getComment` middleware  attaches `comment` instance on request, but in `subscribe` function we need course instance. Earlier, you can se `diff`, also that middleware is skipped for subscribe route, obviously for a reason.